core/src/main/java/org/pentaho/di/connections/ConnectionDetails.java

```diff
@@ -67,10 +67,6 @@ default void setDescription( String description ) {
 
   void setSpace( VariableSpace space );
 
-  default boolean hasBuckets() {
-    return true;
```

Comment:
added during 10.1 to support VFS with this commit - https://github.com/pentaho/pentaho-kettle/commit/1da3c08f00bb6fac2999d08ed35d6cb04fe87dd8

should not go here with other generic connection details information, It should go with VFS specific information